cpp/src/aztec/honk/proof_system/prover.cpp

```diff
@@ -45,6 +45,12 @@ Prover<settings>::Prover(std::shared_ptr<waffle::proving_key> input_key, const t
 // , queue(proving_key.get(), &transcript) // TODO(Adrian): explore whether it's needed
 {}
 
+template <typename settings> void Prover<settings>::reset()
```

Comment:
Is there documentation for the function?

Reply:
This was a copy of a similar function in the plonk prover - thought it was useful but I didn't actually use it so I just deleted it.